framework/src/main/java/org/apache/drill/test/framework/TestDriver.java

```diff
@@ -402,7 +405,7 @@ public void run() {
   private static void runGenerateScript(DataSource datasource) {
     int exitCode = 0;
     String command = CWD + "/resources/" + datasource.src;
-    LOG.debug("Running command " + command);
+    LOG.info("Running command " + command);
```

Comment:
+1 on adding more information. It's really hard to debug when something goes wrong.